SUS-5506 | improve Close/maintenance script

hakubo opened the pull request:
https://wikia-inc.atlassian.net/browse/SUS-5506

hakubo committed:
unify logging on Close/maintenance

hakubo committed:
missing changes

macbre commented on extensions/wikia/WikiFactory/Close/maintenance.php:
```diff
@@ -90,13 +91,11 @@ public function execute() {
-		$this->log( $dbr->lastQuery() );
-
-		$this->log( 'Will start in 5 seconds...' );
+		$this->info( 'Will start in 5 seconds...' );
```
Does it make sense to log this line to Kibana? I'd actually remove this sleep. This script is no longer run manually, so we  do not need to wait for anyone :)

macbre committed:
Merge branch 'dev' into SUS-5506

macbre committed:
CloseWikiMaintenance - remove initial sleep

macbre committed:
CloseWikiMaintenance | improve exceptions logging

macbre committed:
CloseWikiMaintenance: some code linting

macbre committed:
Archive_Tar class is now loaded via composer's autoloader

macbre committed:
WikiFactory/Close/maintenance.php: doh!

ghost commented:
@macbre - is there any benefit in using `maintenance/Maintenance.php` and

```
$maintClass = 'CloseWikiMaintenance ';
require_once( RUN_MAINTENANCE_IF_MAIN );
```

macbre committed:
WikiFactory/Close/maintenance.php: refer to wiki-factory-close-marked-wikis.yaml

macbre committed:
docker/maintenance/README.md: update kubectl instruction to get all cronJobs

ghost commented:
👍

macbre committed:
SUS-5506 | kill a notice

PHP Notice: Only variables should be passed by reference in extensions/wikia/WikiFactory/Close/maintenance.php on line 335

macbre committed:
CloseWikiMaintenance: debug logs are skipped on production
